docs/references/http-websocket-apis/public-api-methods/account-methods/account_tx.md

```diff
@@ -440,10 +439,7 @@ The response follows the [standard format][], with a successful result containin
 | `ledger_index_max` | Integer - [Ledger Index][] | The ledger index of the most recent ledger actually searched for transactions. |
 | `limit`            | Integer                    | The `limit` value used in the request. (This may differ from the actual limit value enforced by the server.) |
 | `marker`           | [Marker][]                 | Server-defined value indicating the response is paginated. Pass this to the next call to resume where this call left off. |
-| `meta`             | Object (JSON)              | (JSON mode) The transaction results metadata in JSON. |
-| `meta_blob`        | String (Binary)            | (Binary mode) The transaction results metadata as a hex string. |
 | `transactions`     | Array                      | Array of transactions matching the request's criteria, as explained below. |
-| `validated`        | Boolean                    | If included and set to `true`, the information in this response comes from a validated ledger version. Otherwise, the information is subject to change. |
```

Comment:
This line should not be removed. There _is_ a `validated` field at the same level as the `transactions` array in both the v1 and v2 responses, as in this screenshot:
![2025-02-25_141808_174197063](https://github.com/user-attachments/assets/d6e16588-01d8-4dbb-bb65-742049660acc)

Reply:
Ah, it appears to be a rippled vs Clio difference

Reply:
What is meaning the inner "validated" ? It looks redundant to me.